src/evict/evict_lru.c

```diff
@@ -798,13 +798,20 @@ __evict_update_work(WT_SESSION_IMPL *session, bool *eviction_needed)
      * because the cost of retrieving a recently reconciled page is higher in that configuration. In
      * the local storage case scrub dirty pages and keep them in cache if we are less than half way
      * to the clean, dirty and updates triggers.
+     *
+     * There's an experimental flag WT_CACHE_EVICT_SCRUB_UNDER_TARGET that can be turned on to
+     * enable scrub eviction as long as cache usage overall is under half way to the trigger limit.
      */
     if (__wt_conn_is_disagg(session) && bytes_inuse < (uint64_t)(trigger * bytes_max) / 100)
         LF_SET(WT_EVICT_CACHE_SCRUB);
     else if (bytes_inuse < (uint64_t)((target + trigger) * bytes_max) / 200) {
-        if (bytes_dirty < (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200 &&
-          bytes_updates < (uint64_t)((updates_target + updates_trigger) * bytes_max) / 200)
+        if (F_ISSET(&conn->cache_eviction_controls, WT_CACHE_EVICT_SCRUB_UNDER_TARGET)) {
```

Comment:
I want the these new optimizations to be enabeld by default unless it is going to impact a wide range of production clusters. We expect it will be improve the situation for many clusters, so enable it by default and whoever faces the problem and they can disable them.

We will note them in the WT release notes and share it with TS whenever we backport these changes.